packages/arb-token-bridge-ui/src/components/common/UnsupportedNetworkContent.tsx

```diff
@@ -0,0 +1,41 @@
+import { useAppState } from '../../state'
+import { networksListArray, networkStyleMap } from '../../util/networks'
+import { Button } from './Button'
+
+export const UnsupportedNetworkContent = () => {
+  const {
+    app: { changeNetwork }
```

Comment:
@spsjvc Can we have a this `changeNetwork` method abstracted and exposed from somewhere else instead of storing this conditionally in app-context?
It will address this bug - If no known network is getting initialized at the app start, the `changeNetwork` method is not working.